feat: basic course part-2 (#20)
Flouse/ckb-academy: src/content/courses/basic-operation/chapter_1/chapter_1.mdx

<br />
<br />

# Blocks

Each square below represents a block, displaying the latest nine blocks on the CKB testnet. The noteworthy information in each block is the transactions displayed in the lower half.

<WalletReady>
  <BlocksSection />
</WalletReady>

When you click on any of the transactions, you will see detailed information in JSON format. You may notice that a real transaction is slightly different from the structure we learned in the theory classes. To be precise, it has become more complex, but it's still graspable if you remember the structure:  `input => output`

If you don't understand what's going on in all these fields for the moment, don't worry, we'll later break it down one by one.

<br />
# Testnet Chain
It is also important to know the configuration information of this test chain.

<WalletReady>
  <TestnetConfigSection />
</WalletReady>

`PREFIX：ckt` indicates that this chain is the test chain rather than the mainnet.

> `SCRIPTS` represent the built-in smart contracts, i.e. some of the system's built-in locks that type and lock can use.

Reviewer: ```suggestion
`SCRIPTS` represent the built-in smart contracts, i.e. some of the system's built-in locks that type and lock scripts can use.
```

> Several built-in smart contracts will be pre-deployed in each CKB chain's genesis block. The following are three of the built-in smart contracts.

Reviewer: ```suggestion
Several built-in smart contracts was pre-deployed in CKB testnet's genesis block. The following are three of the built-in smart contracts.
```

- SECP256K1_BLAKE160: the system's default smart contract for the lock script of cells, used to protect the ownership of the cells.
- SECP256K1_BLAKE160_MULTISIG: the multi-signature version of SECP256K1_BLAKE160.
- DAO: NervosDAO contract. Feel free to ignore it if you don't understand.

---

> #### The above information is what we must understand.

Reviewer: ```suggestion
Well, that's all the information we need to know now.
```
